modules/veteran/app/models/veteran/service/representative.rb

```diff
@@ -111,6 +112,29 @@ def diff(rep_data)
         end
       end
 
+      def self.get_suffix(first_name, last_name)
+        first_suffix = first_name.split.last if first_name.split.count > 1
+        last_suffix = last_name.split.last if last_name.split.count > 1
+        [first_suffix, last_suffix]
+      end
+
+      def self.get_representatives(first_name, last_name)
+        suffix = get_suffix(first_name, last_name)
+
+        representatives = where('lower(first_name) = ? AND lower(last_name) = ?', first_name&.downcase,
+                                last_name&.downcase)
+
+        if representatives.blank? && suffix.present?
```

Comment:
I think we want `suffix.any?` since `get_suffix` returns `[nil, nil]` if there are no suffixes, and `[nil, nil].present?` is `true`.